Ships as Raw.txt

```diff
@@ -0,0 +1,780 @@
+Zoom 93% 
+
+Name|Gladiator I-class Star Destroyer
+Faction|Empire 
+Size|Small
+Points|56
+Hull|5
+Defense token|Br Rd Ev
+Command|2 
+Squad|2 
+Eng|3 
+Nav chart 
+Speed 1|2 
+Speed 2|1 1 
+Speed 3|0 1 1
+Front Arc 3 |RRKK 
+Side Arc 2 |KKKK
+Rear Arc 1 |RK
+Anti Squadron|B
+Upgrade|Title Officers Weapon Team Support Ordance
+FrontArcOffset|8.7
+//How far from the front corner does the hull zone line hit the cardboard edge//
+RearArcOffset|7.5
+//How far from the rear corner does the hull zone line hit the cardboard edge//
+FrontArcConjunction|33
+//How far from the front of the ship do the front hull zone lines meet//
+RearArcConjunction|37
+//How far from the rear of the ship do the rear hull zone lines meet//
+
+Name|Gladiator II-class Star Destroyer
+Faction|Empire 
+Size|Small
+Points|62
+Hull|5
+Defense token|Br Rd Ev
+Command|2 
+Squad|2 
+Eng|3 
+Nav chart 
+Speed 1|2 
+Speed 2|1 1 
+Speed 3|0 1 1
+Front Arc 3 |RRKK 
+Side Arc 2 |KKKR
+Rear Arc 1 |RK
+Anti Squadron|BB
+Upgrade|Title Officers Weapon Team Support Ordance
+FrontArcOffset|8.7
+//How far from the front corner does the hull zone line hit the cardboard edge//
+RearArcOffset|7.5
+//How far from the rear corner does the hull zone line hit the cardboard edge//
+FrontArcConjunction|33
+//How far from the front of the ship do the front hull zone lines meet//
+RearArcConjunction|37
+//How far from the rear of the ship do the rear hull zone lines meet//
+
+
+Name|Consular Charger C70
+Keyword|Clone
+Faction|GAR
+Size|Small
+Points|45
+Hull|4
+Defense token|Con Rd Ev
+Command|1 
+Squad|1
+Eng|2
+Nav chart 
+Speed 1|2
+Speed 2|2 1 
+Speed 3|1 1 1
+Speed 4|1 0 1 1 
+Front Arc 2 |RRB
+Side Arc 2 |RR
+Rear Arc 1 |RB
+Anti Squadron|K
+Upgrade|Title Officers Support TurboLaser Offensive   
+FrontArcOffset|15.1
+//How far from the front corner does the hull zone line hit the cardboard edge//
+RearArcOffset|5.2
+//How far from the rear corner does the hull zone line hit the cardboard edge//
+FrontArcConjunction|41
+//How far from the front of the ship do the front hull zone lines meet//
+RearArcConjunction|29
+//How far from the rear of the ship do the rear hull zone lines meet//
+
+
+
+Name|Consular-class Armed Cruiser
+Keyword|Senate Comms
+Faction|GAR
+Size|Small
+Points|37
+Hull|4
+Defense token|Con Rd Ev
+Command|1 
+Squad|1
+Eng|2
+Nav chart 
+Speed 1|2
+Speed 2|2 1 
+Speed 3|1 1 1
+Speed 4|1 0 1 1 
+Front Arc 2 |BBK
+Side Arc 2 |BK
+Rear Arc 1 |B
+Anti Squadron|bb
+Upgrade|Title Support Defense Ordnance Officer
+FrontArcOffset|15.1
+//How far from the front corner does the hull zone line hit the cardboard edge//
+RearArcOffset|5.2
+//How far from the rear corner does the hull zone line hit the cardboard edge//
+FrontArcConjunction|41
+//How far from the front of the ship do the front hull zone lines meet//
+RearArcConjunction|29
+//How far from the rear of the ship do the rear hull zone lines meet//
+
+Name|Acclamator II

```

Comment:
I'm fairly sure the card says something like assault cruiser in addition to this.